source/documentation/retire-packages.html.md

```diff
@@ -0,0 +1,50 @@
+---
+author: ykarel
+title: Retiring a package
+---
+
+# Retiring a package from RDO
+
+1. toc
+{:toc}
+
+### How to remove an OpenStack package from RDO Trunk
+
+Package for a project can exist across different releases. So retiring it in RDO needs to go via stages,
```

Comment:
We should allow retire packages *only from on-development release* not after GA.

Reply:
Added a NOTE